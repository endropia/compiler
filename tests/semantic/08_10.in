var
	a: integer;
begin
	read('aaaaaaaaaaaaaa');
end.