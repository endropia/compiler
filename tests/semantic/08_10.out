(4, 7) Rvalue can not be used in read procedure